### PR TITLE
Gromacs

### PR DESCRIPTION
**Pull request by fbaru-dev**
*Issue #5 , if available:*

*Description of changes:*
Added the Gromacs solution to the project. Deploying, testing, job submission and resources deletion have been tested.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/gromacs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Compiled source #
```
Comment (mhuguesaws) [inline apps/gromacs/.gitignore:1]: Remove this file

File: apps/gromacs/.pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# See https://pre-commit.com for more information
```
Comment (mhuguesaws) [inline apps/gromacs/.pre-commit-config.yaml:1]: Remove this file

File: apps/gromacs/amis/amzn2-pc-gromacs/amzn2-pc-gromacs.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  "provisioners": [
+    {
+      "type": "shell",
+      "inline": [
+        "sudo yum -y update --security",
+        "sudo yum remove -y dpkg"
+      ]
+    },
+    {
+      "type": "shell",
```
Comment (mhuguesaws) [inline apps/gromacs/amis/amzn2-pc-gromacs/amzn2-pc-gromacs.json:105]: Remove the modules install. It is already part of the Intel oneAPI install.

File: scripts/install/modules_install.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (mhuguesaws) [inline scripts/install/modules_install.sh:1]: Remove the modules_install script.

**Review by mhuguesaws — COMMENTED** [review]
Please address the comments.

File: apps/gromacs/.gitignore (hunk 1 above)
Comment (fbaru-dev) [reply apps/gromacs/.gitignore:1]: done

File: apps/gromacs/.pre-commit-config.yaml (hunk 2 above)
Comment (fbaru-dev) [reply apps/gromacs/.pre-commit-config.yaml:1]: done

File: apps/gromacs/amis/amzn2-pc-gromacs/amzn2-pc-gromacs.json (hunk 3 above)
Comment (fbaru-dev) [reply apps/gromacs/amis/amzn2-pc-gromacs/amzn2-pc-gromacs.json:105]: done

File: apps/gromacs/docs/packer_ami_variables.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+Each OS folder has in their respective `variables.json` file that holds the same variables.
+
+Here is a description of the variables and their default value:
+
+| Name                       | Description                                      | Type    | Default                | Required |
+| -------------------------- | ------------------------------------------------ | ------- | ---------------------- | -------- |
+| application\_name          | Application name                                 | String  | gromacs                | no       |
+| application\_version       | Gromacs version                                  | String  | v2021.4                | no       |
+| aws\_region                | AWS Region where packer will create the instance | String  | us-east-2              | no       |
```
[the remark this answers is not in the document]
Comment (fbaru-dev) [reply apps/gromacs/docs/packer_ami_variables.md:14]: done

File: scripts/install/modules_install.sh (hunk 4 above)
Comment (fbaru-dev) [reply scripts/install/modules_install.sh:1]: done

File: apps/gromacs/performance/gromacs-download-data.sh [hunk 6]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (fbaru-dev) [reply apps/gromacs/performance/gromacs-download-data.sh:1]: done

**Review by mhuguesaws — APPROVED** [review]
LGTM.

Thank you for the your contribution @fbaru-dev

**Pull request merged**